application/src/main/java/org/opentripplanner/apis/gtfs/mapping/routerequest/RouteRequestMapper.java

```diff
@@ -41,11 +41,11 @@ public static RouteRequest toRouteRequest(
     request.setFrom(parseGenericLocation(args.getGraphQLOrigin()));
     request.setTo(parseGenericLocation(args.getGraphQLDestination()));
     request.setLocale(GraphQLUtils.getLocale(environment, args.getGraphQLLocale()));
-    if (args.getGraphQLSearchWindow() != null) {
-      request.setSearchWindow(
-        DurationUtils.requireNonNegativeMax2days(args.getGraphQLSearchWindow(), "searchWindow")
-      );
-    }
+    request.setSearchWindow(
+      args.getGraphQLSearchWindow() != null
+        ? DurationUtils.requireNonNegativeMax2days(args.getGraphQLSearchWindow(), "searchWindow")
```

Comment:
Consider: Utility functions should be generic - instead of having a limit as part of the name, the limit should be passed i: `DurationUtils.requireNonNegativeWithMaxLimit(args.getGraphQLSearchWindow(), "searchWindow", TWO_DAYS)`.

Reply:
I think we have had this discussion once or twice before. The reason why I originally added these methods was that I didn't want to end up in a situation where each argument had a arbitrarily different limit. I don't have a strong opinion on this though so if you want, I can change this.